### PR TITLE
Feat/Based styled components and icons

### PR DESCRIPTION
**Pull request by Int-Angel**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Installed dependencies:
- Styled components
- Carbon icons
- And created based styled components

* **What is the current behavior?** (You can also link to an open issue here)
- Ther is no current behabior

* **What is the new behavior (if this is a feature change)?**
-  new components and dependencies

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
*
no

* **Other information**:

### CODE REVIEW

**Review by Jhoscar22 — APPROVED** [review]

File: .DS_Store
Comment (antonieto) [inline .DS_Store:1]: .DS_Store :c

File: data-dashboard/lib/components/BoardList/BoardList.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+const BoardListComponent = styled.div`
+  width: 100%;
+  height: 100%;
+
+  display: flex;
+  flex-direction: column;
+`;
+
+export default function BoardList() {
+  const boards = [
```
Comment (antonieto) [inline data-dashboard/lib/components/BoardList/BoardList.tsx:24]: Please move this to a constant named `MOCK_BOARD_LIST`

File: yarn.lock
Comment (antonieto) [inline yarn.lock:1]: This yarn.lock file shouldn't exist

**Review by antonieto — CHANGES_REQUESTED** [review]
Overall good job, we ne need to get rid of those files though.

File: data-dashboard/lib/components/BoardList/BoardList.tsx (hunk 1 above)
Comment (antonieto) [reply data-dashboard/lib/components/BoardList/BoardList.tsx:24]: Also, make sure to move it out of the component

File: data-dashboard/lib/components/BoardList/BoardList.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+const BoardListComponent = styled.div`
+  width: 100%;
+  height: 100%;
+
+  display: flex;
+  flex-direction: column;
+`;
+
+export default function BoardList() {
```
Comment (DavBot02) [inline data-dashboard/lib/components/BoardList/BoardList.tsx:23]: I prefer working with arrow functions but that could be decided by everyone

File: data-dashboard/lib/components/BoardPreview/BoardPreview.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+`;
+
+const BoardPreviewLine = styled.hr`
+  width: 100%;
+  height: 1px;
+  background-color: #8d8d8d;
+  border: none;
+`;
+
+export default function BoardPreview({ name }: Props) {
```
Comment (DavBot02) [inline data-dashboard/lib/components/BoardPreview/BoardPreview.tsx:46]: Might be a good idea to add return types to every function

File: data-dashboard/.eslintrc.json [hunk 4]
```diff
@@ -7,6 +7,9 @@
     "airbnb",
     "airbnb/hooks"
   ],
+  "globals": {
+    "JSX": true
```
Comment (antonieto) [inline data-dashboard/.eslintrc.json:11]: What does this rule do?

File: .gitignore [hunk 5]
```diff
@@ -39,3 +39,5 @@ next-env.d.ts
 
 /node_modules
 /.vscode
+
+.DS_Store
```
Comment (Jhoscar22) [inline .gitignore:43]: Bien

**Review by antonieto — APPROVED** [review]

**Review by Jhoscar22 — APPROVED** [review]

**Pull request merged**